db_test.go

```diff
@@ -1133,7 +1133,7 @@ var _ = Describe("ORM", func() {
 	var db *pg.DB
 
 	BeforeEach(func() {
-		db = pg.Connect(pgOptions())
+		db = pg.Connect(pgOptions()).WithParam("shard", pg.Ident("public"))
```

Comment:
It should be `pg.Safe("public")` - see https://github.com/go-pg/sharding/blob/master/cluster.go#L98-L100

Reply:
Aaaah, OK. I missed that! I'm closing the PR – thanks.